src/docsView.ts

```diff
@@ -232,6 +233,14 @@ export class DocsViewViewProvider implements vscode.WebviewViewProvider {
 	private updateConfiguration() {
 		const config = vscode.workspace.getConfiguration('docsView');
 		this._updateMode = config.get<UpdateMode>('documentationView.updateMode') || UpdateMode.Live;
+
+	}
+
+	private getStyleOverrides() {
+		const config = vscode.workspace.getConfiguration('docsView');
+		const fontSize = config.get<number>('documentationView.fontSize');
```

Comment:
When you get the config here, pass in the current document. This ensures that `config` will look up the correct setting value for the document